packages/client-core/src/systems/ui/MediaControlsUI.tsx

```diff
@@ -87,7 +108,11 @@ const MediaControlsView = (props: MediaControlsProps) => {
             background-color: grey;
         }`}
         </style>
-        {mediaComponent.paused.value ? <PlayArrow style={{ fill: 'white' }} /> : <Pause style={{ fill: 'white' }} />}
+        {mediaComponent.paused.value ? (
+          <PlayArrow style={{ fill: 'white', width: `100%`, height: `100%` }} />
```

Comment:
its the client so not relevant for now, but we need to eventually write these in tailwind, is it possible to write them in tailwind now

Reply:
as you saw in call, tailwind is not currently supported here. added a todo